avocado/core/loader.py

```diff
@@ -430,8 +422,7 @@ def get_decorator_mapping():
                 test.MissingTest: output.TERM_SUPPORT.fail_header_str,
                 BrokenSymlink: output.TERM_SUPPORT.fail_header_str,
                 AccessDeniedPath: output.TERM_SUPPORT.fail_header_str,
-                test.Test: output.TERM_SUPPORT.healthy_str,
-                FilteredOut: output.TERM_SUPPORT.warn_header_str}
+                test.Test: output.TERM_SUPPORT.healthy_str}
```

Comment:
OK, let's remove dead code, we can always add new filtered type the way it's the most appropriate when we do have the filtering (btw we can filter by method names `avocado list examples/tests/gendata.py:bsod -V` but it does not show FILTERED either)